src/hotspot/share/utilities/globalDefinitions_gcc.hpp

```diff
@@ -61,6 +72,16 @@
 #include <sys/time.h>
 #endif // LINUX || _ALLBSD_SOURCE
 
+#ifdef _AIX
+#if defined(__open_xl_version__)
+  #if __open_xl_version__ < 17
+  #error "open xlc < 17 not supported"
+  #endif
+#else
+  #error "xlc version not supported, macro __open_xl_version__ not found"
+#endif
+#endif // AIX
```

Comment:
Can probably be shortened like this:

```suggestion
#ifdef _AIX
#if !defined(__open_xl_version__) || (__open_xl_version__ < 17)
  #error "this xlc version is not supported"
#endif
#endif // AIX
```

Reply:
followed your proposal.